hw/mcu/nordic/nrf5340/pkg.yml

```diff
@@ -104,3 +104,10 @@ pkg.deps.I2C_3':
 
 pkg.deps.NRF5340_EMBED_NET_CORE:
     - "@apache-mynewt-core/hw/mcu/nordic/nrf5340/net_core_image"
+
+# As per Errata 26 for Engineering RevA silicon we should not
+# depend on CTRL-AP: APPROTECT.DISABLE and SECUREAPPROTECT.DISABLE
+# registers since they are not functional. It causes quite a lot of
+# crashes and hardfaults
+pkg.cflags.MCU_NRF5340_REVA_SILICON:
```

Comment:
I'd not make this as revA workaround but rather a general config as provided by nrfx:  enabled, user controlled and UICR controlled.

Reply:
done.